packages/select/src/FilterMultiSelect/components/ListBox/ListBox.module.scss

```diff
@@ -7,7 +7,7 @@
   padding-right: $spacing-sm;
   margin: 0 $spacing-sm 0 0;
   display: grid;
-  max-height: 210px;
+  max-height: 344px;
```

Comment:
Actually, would it make sense to have this scale with zoom? I'm assuming the text inside will scale and then eventually get crushed inside the max height... Thoughts @dougmacknz?

Reply:
Ah the age old question. Yeah I think I'd go for rems here. Some thoughts on this topic here https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/#widths-and-heights

Reply:
Yeah great article